Simplenote/Classes/NoteOptionsViewController.swift

```diff
@@ -1,14 +1,39 @@
 import UIKit
+import CoreSpotlight
 
 /// A class used to display options for the note that is currently being edited
-class NoteOptionsViewController: UITableViewController {
+final class NoteOptionsViewController: UITableViewController {
 
     /// Array of `Section`s to display in the view.
     /// Each `Section` has `Rows` that are used for display
     fileprivate var sections: [Section] {
         return [optionsSection, linkSection, collaborationSection, trashSection]
     }
 
+    /// The note from the editor that we will change settings for
+    private var note: Note
+
+    /// The delegate to notify about
+    /// chaanges made here
+    weak var delegate: NoteOptionsViewControllerDelegate?
+
+    /// Formats number of collaborators to respect locales
+    private var collaboratorNumberFormatter = NumberFormatter()
+
+    /// Activity indicator that displays when note is publishing or unpublishing
+    private var publishActivityIndicator = UIActivityIndicatorView(style: SPUserInterface.isDark ? .white : .gray)
```

Comment:
Would this one react to traits changes?

Reply:
Updated to use a colour now that will switch with themes.